src/OrbitQt/CaptureOptionsDialog.ui

```diff
@@ -544,16 +544,76 @@ p, li { white-space: pre-wrap; }
           <item>
            <widget class="QCheckBox" name="threadStateChangeCallstackCollectionCheckBox">
             <property name="toolTip">
-             <string>This allows capturing call stacks when threads change their states</string>
+             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;This allows capturing callstacks when threads become runnable or stop running.&lt;/p&gt;&lt;p&gt;Collecting callstacks on thread state changes introduces significant performance overhead. Consider reducing the &lt;span style=&quot; font-style:italic;&quot;&gt;raw stack size to copy&lt;/span&gt; below.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
             <property name="text">
-             <string>Collect call stacks on thread state changes</string>
+             <string>Collect callstacks on thread state changes (expensive) ⓘ</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
+          <item>
+           <widget class="QWidget" name="threadStateChangeCallstackMaxCopyRawStackSizeWidget" native="true">
+            <layout class="QHBoxLayout" name="horizontalLayout">
+             <property name="leftMargin">
+              <number>20</number>
+             </property>
+             <property name="topMargin">
+              <number>0</number>
+             </property>
+             <property name="rightMargin">
+              <number>0</number>
+             </property>
+             <property name="bottomMargin">
+              <number>0</number>
+             </property>
+             <item>
+              <widget class="QLabel" name="threadStateChangeCallstackMaxCopyRawStackSizeLabel">
+               <property name="toolTip">
+                <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;In order to compute the callstack on thread state changes, we need to copy a portion of the raw stack. Specify how much data Orbit should maximally copy. A smaller value reduces the number of computed call frames, but also reduces the performance overhead.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
+               </property>
+               <property name="text">
+                <string>Max raw stack size to copy (multiple of 8 Bytes):</string>
```

Comment:
Nit: 
I'd suggest to just go with
`<string>Max raw stack size to copy in bytes:</string>`
looking at the screenshot I asked myself whether we copy 256 bytes or 2048 = 8*256 bytes. Also this doesn't really matter that much from a users perspective I think just enforcing this via the control is easier.

Reply:
Just to note: We already have the exact same text above in the frame-pointer case. The original text was modified quite often, also with UX. Should I still change it?